Github/legos/github_list_stale_issues/README.md

```diff
@@ -0,0 +1,28 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h2>Github List Stale Issues</h2>
+
+<br>
+
+## Description
+This Lego lists all stale issues in a Github Repository
+
+## Lego Details
+
+    github_list_pull_requests(handle: object , owner:str, repository:str, age_to_stale:int)
+
+        handle: Object of type unSkript Github Connector
+        owner: String, Username of the GitHub user. Eg: "johnwick"
+        repository: String, Full name of the GitHub repository. Eg: "unskript/Awesome-CloudOps-Automation"

```

Comment:
There should be **Full path of the GitHub repository** instead of **Full name of the GitHub repository**

Reply:
Please check now